doc/releases/nautilus.rst

```diff
@@ -216,6 +216,13 @@ Instructions
         "ceph version 14.2.0 (...) nautilus (stable)": 22,
      }
 
+#. Scan for any OSDs deployed with the old ceph-disk tool to ensure
```

Comment:
Will an osd created by ceph-disk start in nautilus? You might need to run this step after installing the new packages but before restarting the osd daemons.

Reply:
it worked for me.  i think it's fine because all of hte /var/lib/ceph/osd/* directories are mounted and systemd units already active.. so it's just a restart of the current systemd units